client-go/cmd/client/client.go

```diff
@@ -47,7 +80,43 @@ func main() {
 		log.Fatalln(errors.Wrap(err, "failed to connect to service"))
 	}
 
-	consumer := consumer.New(conn)
+	ticker := time.NewTicker(time.Millisecond * time.Duration(config.RequestIntervalMS))
+	defer ticker.Stop()
+	done := make(chan bool)
+	for {
+		select {
+		case <-done:
+			return
+		case <-ticker.C:
+			// create a new client for each request
+			consumer := consumer.New(conn)
+			// create random delay to simulate processing load on server
+			// no need for crypto secure rand with seed
+			delay := rand.Intn(config.PublishDelayMaxMS)
```

Comment:
Generate a random millisecond value to send to the service to simulate async processing load. It would be easier to generate this on the server side, but I chose to do it in the client in order to demonstrate modification and regeneration of the grpc / protobuf definitions.